charts/incubator/plik/values.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  repository: tccr.io/truecharts/plik
+  pullPolicy: IfNotPresent
+  tag: 1.3.6@sha256:f9c31219bfb0793c3b9ce4ec304cd52c4dfef106ae452ab0f6a57a0a0b575fbd
+
+securityContext:
+  readOnlyRootFilesystem: false
+
+service:
+  main:
+    ports:
+      main:
+        protocol: HTTP
+        targetPort: 8080
+        port: 10599
+
+persistence:
+  data:
+    enabled: true
+    mountPath: /data
+  plikd:
+    enabled: true
+    mountPath: /home/plik/server/plikd.cfg
```

Comment:
You **cannot** mount files like that.
Only with configmap/secrets

Reply:
https://github.com/root-gg/plik/blob/master/server/plikd.cfg

Reply:
thats a lot of options 😵‍💫 but ill get to it.

Reply:
Start with a few jus to get it working and then you can start adding more